### PR TITLE
feat: add Boba Eth Mainnet

### PR DESCRIPTION
**Pull request by boyuan-chen**
### Testing

```
~ ❯ hyperlane send message --relay                                                              8s 10:30:39 AM
Hyperlane CLI
? Select network type Testnet
? Select the origin chain holesky
? Select network type Mainnet
? Select the destination chain boba
Running pre-flight checks for chains...
✅ Chains are valid
✅ Signer is valid
WARNING: 0x239cD87712B2CB0c35f1DEAf4FB68B8F9585e9D1 has low balance on holesky. At least 0.0017731781112 ETH recommended but found 0.0 ETH
? Deployment may fail due to insufficient balance(s). Continue? yes
Dispatching message
Pending https://holesky.etherscan.io/tx/0x48f69da6b2336613f679f14e6fff9a1ff9cbc050e3eb22b1f6a8c16127ed8465 (waiting 1 blocks for confirmation)
Sent message from holesky to 0xF8f6976CDDdE253855552191c533466221dD54db on boba.
Message ID: 0x3f2ce4c24d1d15b43b1df5069683e0554a49e8aeb4a4693afc2d37423c100895
Message:
    parsed:
      version: 3
      nonce: 416
      origin: 17000
      sender: "0x000000000000000000000000239cd87712b2cb0c35f1deaf4fb68b8f9585e9d1"
      destination: 288
      recipient: "0x000000000000000000000000f8f6976cddde253855552191c533466221dd54db"
      body: "0x48656c6c6f21"
      originChain: holesky
      destinationChain: boba
    id: "0x3f2ce4c24d1d15b43b1df5069683e0554a49e8aeb4a4693afc2d37423c100895"
    message: "0x03000001a000004268000000000000000000000000239cd87712b2cb0c35f1deaf4\
      fb68b8f9585e9d100000120000000000000000000000000f8f6976cddde253855552191c53346\
      6221dd54db48656c6c6f21"

Unhandled error case in combined provider error handler
Unhandled error case in combined provider error handler
Unhandled error case in combined provider error handler
Attempting self-relay of message
Preparing to relay message 0x3f2ce4c24d1d15b43b1df5069683e0554a49e8aeb4a4693afc2d37423c100895
Unhandled error case in combined provider error handler
Unhandled error case in combined provider error handler
Unhandled error case in combined provider error handler
{
  message: {
    id: '0x3f2ce4c24d1d15b43b1df5069683e0554a49e8aeb4a4693afc2d37423c100895',
    message: '0x03000001a000004268000000000000000000000000239cd87712b2cb0c35f1deaf4fb68b8f9585e9d100000120000000000000000000000000f8f6976cddde253855552191c533466221dd54db48656c6c6f21',
Add Boba Eth Mainnet
    parsed: {
      version: 3,
      nonce: 416,
      origin: 17000,
      sender: '0x000000000000000000000000239cd87712b2cb0c35f1deaf4fb68b8f9585e9d1',
      destination: 288,
      recipient: '0x000000000000000000000000f8f6976cddde253855552191c533466221dd54db',
      body: '0x48656c6c6f21'
    }
  },
  metadata: '0x'
} Relaying message 0x3f2ce4c24d1d15b43b1df5069683e0554a49e8aeb4a4693afc2d37423c100895
Pending https://bobascan.com/tx/0x120b6250854131c7f476800d2c1b9dfcc56d565e9581c228d407afaf29847973 (waiting 1 blocks for confirmation)
Message was self-relayed!
```

### CODE REVIEW

**Comment by JunayedMh** [comment]
You didn’t add the logo.svg file...add the chain logo on your folder,also add changeset file by clicking this option edit the patch with minor
Screenshot_2024-10-12-19-54-16-449_com android chrome-edit

File: chains/boba/logo.svg [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+				<polygon class="st5" points="348.2,343.7 227.7,543.6 348.2,614.9 348.2,488.9 				"/>
+				<polygon class="st6" points="348.2,637.7 346.7,639.5 346.7,731.9 348.2,736.3 468.8,566.5 				"/>
+				<polygon class="st5" points="348.2,736.3 348.2,637.7 227.7,566.5 				"/>
+				<polygon class="st7" points="348.2,614.9 468.7,543.6 348.2,488.9 				"/>
+				<polygon class="st8" points="227.7,543.6 348.2,614.9 348.2,488.9 				"/>
+			</g>
+		</g>
+	</g>
+</g>
+</svg>
```
Comment (xeno097) [inline chains/boba/logo.svg:41]: Hey @boyuan-chen thank you for your PR to the Hyperlane Registry! Can you please crop and minify the SVG file to conform to the standard used in the repo? You can use these tools to do it:
- crop: https://svgcrop.com/
- minify: https://jakearchibald.github.io/svgomg/

File: chains/boba/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+domainId: 288
+isTestnet: false
+name: boba
+nativeToken:
+  decimals: 18
+  name: Ether
+  symbol: ETH
+protocol: ethereum
+rpcUrls:
+  - http: https://mainnet.boba.network
```
Comment (xeno097) [inline chains/boba/metadata.yaml:18]: Here the `deployer` information is missing as an example you can reference one of Hyperlane deployed [chains definitions](https://github.com/hyperlane-xyz/hyperlane-registry/blob/main/chains/arbitrum/metadata.yaml)

**Review by xeno097 — CHANGES_REQUESTED** [review]

File: chains/boba/metadata.yaml (hunk 2 above)
Comment (boyuan-chen) [reply chains/boba/metadata.yaml]: Fixed!

File: chains/boba/logo.svg (hunk 1 above)
Comment (boyuan-chen) [reply chains/boba/logo.svg]: Fixed!

File: .changeset/tiny-clocks-heal.md
Comment (atuser14) [inline .changeset/tiny-clocks-heal.md:1]: - patch
+ minor

**Review by jmrossy — APPROVED** [review]

**Pull request merged**